### PR TITLE
Add bulk actions in content-manager

### PR DESCRIPTION
**Pull request by soupette**
My PR is a:
💅 Enhancement

Main update on the:
Plugin: `content-manager`

This PR adds bulk actions in the content manager plugin

### CODE REVIEW

File: packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          {message => <span className={styles.deleteAll} onClick={onToggleDeleteAll}>{message}</span>}
+        </FormattedMessage>
+      </td>
+    </tr>
+  );
+}
+
+TableDelete.defaultProps = {
+  colspan: 0,
+  number: [],
```
Comment (kamalbennani) [inline packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js]: Did you mean 0 ?

**Comment by mnlbox** [comment]
@soupette Please add some screenshot here. I can't wait until see this awesome feature :wink:

**Comment by lauriejim** [comment]
screen shot 2018-06-12 at 10 37 13 am

File: packages/strapi-plugin-content-manager/services/ContentManager.js [hunk 2]
```diff
@@ -159,4 +159,51 @@ module.exports = {
...
+
+    const filters = strapi.utils.models.convertParams(model, { [`${primaryKey}_in`]: toRemove });
+    const entries = await strapi.query(model, source).find({ where: filters.where }, null, true);
+    const associations = strapi.query(model, source).associations;
+
+    for (let i = 0; i < entries.length; ++i) {
+      const entry = entries[i];
+
+      associations.forEach(association => {
+        if (entry[association.alias]) {
```
Comment (lauriejim) [inline packages/strapi-plugin-content-manager/services/ContentManager.js]: What about morph relations ?

**Pull request merged**